K2Bridge/Models/Request/Aggregations/Aggregation.cs

```diff
@@ -14,7 +14,15 @@ namespace K2Bridge.Models.Request.Aggregations
     /// </summary>
     internal abstract class Aggregation : KustoQLBase, IVisitable
     {
-        public string Key { get; set; }
+        /// <summary>
+        ///  Key of the aggregation.
+        /// </summary>
+        public virtual string Key { get; set; }
+
+        /// <summary>
+        /// Aggregation field.
+        /// </summary>
+        public virtual string Field { get; set; }
```

Comment:
I understand the purpose of this change. And you're right, in the current implemented aggregations, all of them are sharing Field property. BUT, it's not the case for all bucket aggregations. For instance, if we implement Filters bucket aggregations, the field property is not supported (filters is used instead). It's also the case for additional ones not used by visualization (MultiTerms, Sampler, AdjancencyMatrix, ...). I suggest to rollback this change and keep Field as a top object property.

Reply:
For reference, only Metric Aggregations are all supporting Field property.

Reply:
Let me know if this is ok

Reply:
Yes. All good for me. Thanks.